SampleShared/Samples/TextPanel/Runtime Information.prefab.meta

```diff
@@ -1,8 +1,7 @@
 fileFormatVersion: 2
```

Comment:
not sure why this file rename is from material to prefab. Is this accurate?

Reply:
With all the meta files moving and looking very similar, I think git is getting confused with which file became which. The files are fine, they just look odd in this display.